packages/schemas/jest.config.ts

```diff
@@ -0,0 +1,7 @@
+import { merge, Config } from '@silverhand/jest-config';
+
+const config: Config.InitialOptions = merge({
+  testEnvironment: 'node',
```

Comment:
the default value is node?

Reply:
+1

Reply:
Yes. Improved

The `jest.config.ts` in other packages are almost similar (`merge({testEnvironment: 'node'})`), and we'd better simplify them as well.